### DISCUSSION TITLE
Self hosted server

### DISCUSSION THREAD

**Discussion by AlissonSteffens** [mozilla/hubs#4291] [Hubs Cloud Questions] [answered]
Hello. I have a question. Is that a way for me to run the server in my local server? Without having to use Digital Ocean or AWS?

**Comment by emclaren** [comment, marked as answer]
Please see: https://github.com/mozilla/hubs/discussions/3554

**Comment by albirrkarim** [comment]
I think my [article about hosting hubs on VPS](https://github.com/albirrkarim/mozilla-hubs-installation-detailed/blob/main/VPS_FOR_HUBS.md) can help you

**Reply by yengalvez** [reply]
Thanks albirrkarim, I see that the tutorial is not yet finished. When would you have it finished? I would like to try it myself.

**Reply by albirrkarim** [reply]
Sometimes there's a step that I'm not writing down on my tutorial. you know some little hack that we sometimes do but we forget about it hahaha. 

Btw it's already finished.  You can give it a try. and give me messages if there is a missing step/bug that I forget to write it down.

**Reply by yengalvez** [reply]
Amazing! Thanks for your work :D
